packages/client/hmi-client/src/components/tabs/Tab.vue

```diff
@@ -0,0 +1,103 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+import IconClose from '@carbon/icons-vue/es/close/16';
+
+const props = defineProps({
+	name: {
+		type: String,
+		required: true,
+		default: 'New tab'
+	},
+	index: {
+		type: Number,
+		required: true,
+		default: 0
```

Comment:
Aren't `default` and `required` mutually exclusive?

Reply:
Hmm you're right.